shell/platform/android/platform_view_android.cc

```diff
@@ -162,6 +150,20 @@ void PlatformViewAndroid::SurfaceCreated(JNIEnv* env,
   // workaround for https://code.google.com/p/android/issues/detail?id=68174
   fml::jni::ScopedJavaLocalFrame scoped_local_reference_frame(env);
 
+  // We have a drawing surface, so swap in a non-Null rasterizer.
+  SetRasterizer(std::make_unique<GPURasterizer>(nullptr));
+
+  // Eagerly setup the IO thread context. We have already setup the surface.
+  SetupResourceContextOnIOThread();
```

Comment:
This can be moved back into PlatformViewAndroid::Attach

Reply:
Not sure I follow you. I guess it looks like the GPURasterizer acts like the NullRasterizer when there isn't a surface, but is there some reason not to make it explicit? Is there a problem with swapping to the GPURasterizer in the surfaceCreated callback?

Reply:
SetupResourceContextOnIOThread does not require an on-screen surface and can be done during PlatformViewAndroid::Attach

Reply:
Oh, okay. Thought 'this' meant the whole block. Done.